tests/analytics.bats

```diff
@@ -51,15 +56,6 @@ setup() {
   [ "$status" -eq 0 ]
 }
 
-@test "[analytics] Set no auto-create on Solr" {
```

Comment:
We are removing this for suggester to work, correct?

Reply:
Not exactly. We are removing this because the same tests exist(ed) on L155. The key change was setting `"update.autoCreateFields":"false"`  here bin/gxa-index-set-no-autocreate.sh to `false`